bot/slack/commands/ip_whois.py

```diff
@@ -0,0 +1,31 @@
+import netaddr
+from ipwhois import IPWhois
+
+
+def is_ip(ip):
+    try:
+        netaddr.IPNetwork(ip)
+        return True
+    except Exception:
+        return False
+
+
+class command():
```

Comment:
As before, class names should be in `PascalCase`.

Reply:
Fixed in https://github.com/mozilla/MozDef/pull/1018/commits/458723466763d1f51f2879147d4c1c255c4ae221